apps/sms/services/js/message_manager.js

```diff
@@ -468,6 +592,16 @@ var MessageManager = {
     return defer.promise;
   },
 
+  /**
+   * This method checks whether message is received from a sender with specified
+   * number or sent to this particular number. (in case of MMS only 1-to-1,
+   * non-group MMS's are counted).
+   * It's used to differentiate from broadcast messages ---> group MMS.
+   * @param {Number} number Phone number.
```

Comment:
Please be consistent in the case for parameter types. So there are two main styles out of there:
* All upper case - ```{Number|String|Boolean}``` and ```{Object|Array}``` - the most preferable.
* Simple types are lower case and objects/arrays/complex types in upper case ```{number|string|boolean}``` and ```{Object|Array}```.